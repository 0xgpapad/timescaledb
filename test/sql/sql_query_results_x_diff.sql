--make sure diff only has explain output not result output
\! diff ../results/sql_query_results_optimized.out ../results/sql_query_results_unoptimized.out 
