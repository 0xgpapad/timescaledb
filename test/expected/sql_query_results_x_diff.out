--make sure diff only has explain output not result output
\! diff ../results/sql_query_results_optimized.out ../results/sql_query_results_unoptimized.out 
11a12
> SET timescaledb.disable_optimizations= 'true';
53,54c54,55
<                                               QUERY PLAN                                              
< ------------------------------------------------------------------------------------------------------
---
>                                    QUERY PLAN                                   
> --------------------------------------------------------------------------------
56,62c57,62
<    ->  GroupAggregate
<          Group Key: (date_trunc('minute'::text, _hyper_1_0_replica."time"))
<          ->  Result
<                ->  Merge Append
<                      Sort Key: (date_trunc('minute'::text, _hyper_1_0_replica."time")) DESC
<                      ->  Sort
<                            Sort Key: (date_trunc('minute'::text, _hyper_1_0_replica."time")) DESC
---
>    ->  Sort
>          Sort Key: (date_trunc('minute'::text, _hyper_1_0_replica."time")) DESC
>          ->  HashAggregate
>                Group Key: date_trunc('minute'::text, _hyper_1_0_replica."time")
>                ->  Result
>                      ->  Append
64,65d63
<                      ->  Sort
<                            Sort Key: (date_trunc('minute'::text, _hyper_1_1_0_partition."time")) DESC
67,68c65,66
<                      ->  Index Scan using "1-time_plain" on _hyper_1_1_0_1_data
< (13 rows)
---
>                            ->  Seq Scan on _hyper_1_1_0_1_data
> (10 rows)
96,102c94,99
<    ->  GroupAggregate
<          Group Key: (date_trunc('minute'::text, _hyper_1_0_replica."time"))
<          ->  Result
<                ->  Merge Append
<                      Sort Key: (date_trunc('minute'::text, _hyper_1_0_replica."time")) DESC
<                      ->  Sort
<                            Sort Key: (date_trunc('minute'::text, _hyper_1_0_replica."time")) DESC
---
>    ->  Sort
>          Sort Key: (date_trunc('minute'::text, _hyper_1_0_replica."time")) DESC
>          ->  HashAggregate
>                Group Key: date_trunc('minute'::text, _hyper_1_0_replica."time")
>                ->  Result
>                      ->  Append
105,106d101
<                      ->  Sort
<                            Sort Key: (date_trunc('minute'::text, _hyper_1_1_0_partition."time")) DESC
109,111c104,106
<                      ->  Index Scan using "1-time_plain" on _hyper_1_1_0_1_data
<                            Index Cond: ("time" < 'Wed Dec 31 16:15:00 1969 PST'::timestamp with time zone)
< (16 rows)
---
>                            ->  Seq Scan on _hyper_1_1_0_1_data
>                                  Filter: ("time" < 'Wed Dec 31 16:15:00 1969 PST'::timestamp with time zone)
> (13 rows)
